### PR TITLE
[feature] Add getElapsedSince function

### LINKED ISSUE #4: [feat] GetElapsedSince Function
## What is the new or updated feature that you are suggesting?
- This function returns a string representing the elapsed time since a given target date compared to a reference date.
- ex) `지금`, `5분 전`, `10일 전`
## Why should this feature be included?
- It can be used when need string elapsed from a specific date object

### PR DESCRIPTION
**Pull request by altmit**
## Issue
closes #4

### CODE REVIEW

File: packages/utils/date/getElapsedSince/getElapsedSince.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const languageObj = {
+    kr: {
+      justNow: "방금",
+      minutesAgo: "분 전",
+      hoursAgo: "시간 전",
+      daysAgo: "일 전",
+      monthsAgo: "달 전",
+    },
+    eng: {
+      justNow: "now",
```
Comment (Kakamotobi) [inline packages/utils/date/getElapsedSince/getElapsedSince.ts:38]: I think the value should be `"just now"`

File: packages/utils/date/getElapsedSince/getElapsedSince.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      minutesAgo: "minutes ago",
+      hoursAgo: "hours ago",
+      daysAgo: "days ago",
+      monthsAgo: "months ago",
```
Comment (Kakamotobi) [inline packages/utils/date/getElapsedSince/getElapsedSince.ts:42]: It'd be nice if we can provide accurate plurality as well based on the time value.
Ex: `1 minute ago`, `2 minutes ago`

File: packages/utils/date/getElapsedSince/getElapsedSince.test.ts
Comment (Kakamotobi) [inline packages/utils/date/getElapsedSince/getElapsedSince.test.ts:1]: It'd be great if we can test the "boundaries" between the labels. For example, when the time diff is 59s it should be `"Just now"`, and when the time diff is 60s it should  be `"1 minute ago"`. And the same for between minutes and hours, hours and days, days, and months as well.

Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.test.ts:1]: This a function implemented based on 60 seconds, I don’t think there will be any problems.
But I'll add some tests👌

File: packages/utils/date/getElapsedSince/getElapsedSince.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
+type Props = {
+  targetDate: Date;
+  referenceDate?: Date;
```
[the remark this answers is not in the document]
Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:3]: Thank you for recommending good names👍

File: packages/utils/date/getElapsedSince/getElapsedSince.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  language?: "kr" | "eng";
```
[the remark this answers is not in the document]
Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:4]: I tried to judge regardless of language choice, based on the presence of a customLabels
If there are customLabels, I made it overwrite the existing sentences. Here's the code like this
```ts
  const strings =
    customLanguageObj === undefined ? languageObj[language] : customLanguageObj;
```

File: packages/utils/date/getElapsedSince/getElapsedSince.ts (hunk 2 above)
Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:42]: I think additional work will be needed. Korean isn't a problem, but need to consider how plurals are expressed in other countries and with different custom labels.

File: packages/utils/date/getElapsedSince/getElapsedSince.ts [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  };
+};
+
+/**
+ * This function returns a string representing the elapsed time since a given target date compared to a reference date.
+ *
+ * @param {Object} Props - The properties object.
+ * @param {Date} Props.targetDate - The target date to calculate the elapsed time from.
+ * @param {Date} [Props.referenceDate=new Date()] - The reference date to calculate the elapsed time against. (default: new Date())
+ * @param {string} [Props.language="kr"] - The language to use for the time strings. Choose between "kr" and "eng". (defaults: "kr").
```
[the remark this answers is not in the document]
Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:20]: I think we'll use Korean more often 😂

[the remark this answers is not in the document]
Comment (Kakamotobi) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:20]: I was hoping this library could eventually grow to become a bit more "global" 😅

File: packages/utils/date/getElapsedSince/getElapsedSince.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (Kakamotobi) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:4]: I meant if the user provides `customLabels` which could be in another language, the `language` type would not reflect this. This doesn't really affect the user but the discrepancy irks me a little.

[the remark this answers is not in the document]
Comment (altmit) [reply packages/utils/date/getElapsedSince/getElapsedSince.ts:4]: I got it. I will try to improve 👌

**Review by Kakamotobi — APPROVED** [review]

**Review by Kakamotobi — APPROVED** [review]

**Pull request merged**